### PR TITLE
Enable GitHub actions

### PR DESCRIPTION
**Pull request by bonifaido**
As a secondary/backup CI solution I would like to enable GitHub Actions as well because we recently had many troubles with CircleCI.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
+name: CI
+on: [push]
```
Comment (sagikazarmark) [inline .github/workflows/ci.yml]: Shouldn't this rather be `pull_request`?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.13
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.13
+      id: go
```
Comment (sagikazarmark) [inline .github/workflows/ci.yml]:
```suggestion
```

Not used anywhere

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+      DOCKER_LATEST: 1
+      VERSION: latest
+
+    steps:
+
+    - name: Set up Go 1.13
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.13
+      id: go
```
Comment (sagikazarmark) [inline .github/workflows/ci.yml]:
```suggestion
```

Not used anywhere

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.13
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.13
+      id: go
+
+    - name: Check out code into the Go module directory
```
Comment (sagikazarmark) [inline .github/workflows/ci.yml]:
```suggestion
    - name: Check out code
```

File: .github/workflows/ci.yml [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+    steps:
+
+    - name: Set up Go 1.13
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.13
+      id: go
+
+    - name: Check out code into the Go module directory
```
Comment (sagikazarmark) [inline .github/workflows/ci.yml]:
```suggestion
    - name: Check out code
```

File: .github/workflows/ci.yml [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+        sudo -E minikube start --vm-driver=none --cpus 2 --memory 4096 --kubernetes-version=${K8S_VERSION} --extra-config=kubeadm.ignore-preflight-errors=SystemVerification
+
+    - name: Build Docker images
+      run: |
+        make docker
+        make docker-operator
+        make docker-vault-env
+        make docker-webhook
+
+    - name: Install kubectl, helm and kurun
```
Comment (sagikazarmark) [inline .github/workflows/ci.yml]: These could be separate steps (for future reusability)

File: .github/workflows/ci.yml (hunk 1 above)
Comment (bonifaido) [reply .github/workflows/ci.yml]: Yeah, sure.

**Pull request merged**